### PR TITLE
Create seminar S3E17

### PR DESCRIPTION
**Pull request by qhduan**

### CODE REVIEW

**Comment by qhduan** [comment]
@Duan-JM Sorry, I'm too stupid to forget pushing the commit, it's OK now

File: docs/_posts/2019-12-01-seminar-meeting-minutes-3-17.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+---
+title: 第三季17场 AI ML Club 活动纪要
```
Comment (CyFeng16) [inline docs/_posts/2019-12-01-seminar-meeting-minutes-3-17.md]: Maybe we should add `NLP` in this title.

Comment (qhduan) [reply docs/_posts/2019-12-01-seminar-meeting-minutes-3-17.md]: fixed

File: docs/_posts/2019-12-01-seminar-meeting-minutes-3-17.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+1. 贺新：如何高效阅读论文，深挖
+1. 李重阳：给每个神经元加一个地址
+1. 石珅达：高维空间是否可能距离同构
+1. 卓桓：TreeBank的距离增加一个树
+1. 段清华：TreeBank Loss
+
+## 吐槽
+
+1. 贺新：没有oral
+1. 李重阳：poster时间是否不够（申达：建议考虑如果poster时间不够，就考虑oral）
```
Comment (Duan-JM) [inline docs/_posts/2019-12-01-seminar-meeting-minutes-3-17.md]: Maybe “申达” name is wrong here。

Comment (qhduan) [reply docs/_posts/2019-12-01-seminar-meeting-minutes-3-17.md]: fixed

**Comment by Duan-JM** [comment]
@qhduan LGTM! ~

**Pull request merged**